src/components/PostList/PostList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Post } from '../Post/Post';
+import './PostList.scss';
+import { PostListType } from '../Types';
+
+export const PostList = ({ preparedPosts }) => (
+  <div className="postList">
+    {preparedPosts.map(post => (
+      <Post
+        body={post.body}
+        user={post.user}
+        comments={post.comments}
+        title={post.title}
```

Comment:
or use spread operator

```suggestion
        {...post}
```

Reply:
Okay, but i used this syntax to to understand the readability of the code for me)